sm/sm.h

```diff
@@ -430,7 +430,7 @@ struct m0_sm_state_descr {
 	   be called before state machine with this configuration
 	   can be constructed.
 	 */
-	uint32_t    sd_trans[M0_SM_MAX_STATES];
+	int8_t      sd_trans[M0_SM_MAX_STATES];
```

Comment:
I am not sure if a max value of 254 (not including the 255, -1) is enough.

Reply:
M0_SM_MAX_STATES is 64 atm, so more than enough for now.